package.json

```diff
@@ -45,15 +46,17 @@
     "xxhashjs": "^0.2.2"
   },
   "dependencies": {
-    "rollup": "^2.26.4",
     "@rollup/plugin-commonjs": "^15.0.0",
     "@rollup/plugin-node-resolve": "^9.0.0",
     "aes-js": "^3.1.2",
     "another-json": "^0.2.0",
     "base64-arraybuffer": "^0.2.0",
     "bs58": "^4.0.1",
+    "docdash": "^1.2.0",
```

Comment:
These should be added as `devDependencies` rather than regular dependencies.

Reply:
sure 👍🏽